config.yml

```diff
@@ -43,6 +43,7 @@ CreateAutomaticRectilinearStateVectorFile: true
 nBufferClusters: 8
 BufferDeg: 5
 LandThreshold: 0.25
+OffshoreEmisThreshold: 0
```

Comment:
Can you update the readthedocs page with a corresponding description of this new config variable?

Also, add it to `sanitize_input_yaml.py` since this should be a required variable?

Reply:
Thanks for the reminders! Done.